### PR TITLE
Add return err

### PR DESCRIPTION
**Pull request by dongrie**
Fix add-dir command

### Before
before

### After
after

### CODE REVIEW

File: cmd/chains.go [hunk 1]
```diff
@@ -59,19 +59,19 @@ func filesAdd(ctx *config.Context, dir string) error {
 		byt, err := os.ReadFile(pth)
 		if err != nil {
 			fmt.Printf("failed to read file %s, skipping...\n", pth)
-			continue
+			return err
```
Comment (siburu) [inline cmd/chains.go:62]: Why not convert `fmt.Printf` above into `fmt.Errorf` and return it?
Don't forget to remove "\n" and add the original error using "%v".

Comment (dongrie) [reply cmd/chains.go]: https://github.com/hyperledger-labs/yui-relayer/pull/105/commits/21850879ea0f2ffc0d019cf16f7bec34305c0b5f
Use `fmt.Errorf`

File: cmd/chains.go [hunk 2]
```diff
@@ -58,20 +58,17 @@ func filesAdd(ctx *config.Context, dir string) error {
...
-			continue
+			return fmt.Errorf("failed to unmarshal file %s, error: %v", pth, err)
 		}
 		if err := c.Init(ctx.Codec); err != nil {
 			return err
 		}
 		if err = ctx.Config.AddChain(ctx.Codec, c); err != nil {
-			fmt.Printf("%s: %s\n", pth, err.Error())
-			continue
+			return fmt.Errorf("failed add chain %s, error: %v", pth, err)
```
Comment (bluele) [inline cmd/chains.go:71]: `failed to add chain ...` is correct?

Also, could you add an error msg for `c.Init()` at line 68? @dongrie

Comment (dongrie) [reply cmd/chains.go]: https://github.com/hyperledger-labs/yui-relayer/pull/105/commits/e4c140f59be72296ec2b408165ea317e0746d573
Fixed error message and add error msg to c.Init()

**Review by bluele — APPROVED** [review]
LGTM👍

**Review by siburu — APPROVED** [review]
LGTM, too!

**Pull request merged**